src/main/java/edu/harvard/iq/dataverse/DatasetVersion.java

```diff
@@ -1356,24 +1356,37 @@ public List<DatasetRelPublication> getRelatedPublications() {
                 for (DatasetFieldCompoundValue publication : dsf.getDatasetFieldCompoundValues()) {
                     DatasetRelPublication relatedPublication = new DatasetRelPublication();
                     for (DatasetField subField : publication.getChildDatasetFields()) {
-                        if (subField.getDatasetFieldType().getName().equals(DatasetFieldConstant.publicationCitation)) {
-                            String citation = subField.getDisplayValue();
-                            relatedPublication.setText(citation);
-                        }
-
-                        
-                        if (subField.getDatasetFieldType().getName().equals(DatasetFieldConstant.publicationURL)) {
-                            // We have to avoid using subField.getDisplayValue() here - because the DisplayFormatType 
-                            // for this url metadata field is likely set up so that the display value is automatically 
-                            // turned into a clickable HTML HREF block, which we don't want to end in our Schema.org JSON-LD output.
-                            // So we want to use the raw value of the field instead, with 
-                            // minimal HTML sanitation, just in case (this would be done on all URLs in getDisplayValue()).
+                        switch (subField.getDatasetFieldType().getName()) {
+                        case DatasetFieldConstant.publicationCitation:
```

Comment:
Aren't the `case` statements usually indented?

Reply:
FWIW: We have both styles and the Java recommendation appears to be not to indent since the cases are just labels and not a scope. I'll go ahead and indent.

Reply:
Huh, where do you see that? I see indented at https://docs.oracle.com/javase/tutorial/java/nutsandbolts/switch.html

Anyway, thanks for changing it.

Reply:
I ran across https://www.oracle.com/java/technologies/javase/codeconventions-statements.html#468 and online arguments back and forth.

Reply:
So I see! https://stackoverflow.com/questions/4509039/why-the-strange-indentation-on-switch-statements

I didn't realize not indenting `case` was a thing. I'm just used to what Netbeans does. Sorry for the noise!